src/content/whats-new/2024/10/whats-new-10-10-pathpoint.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 'Connect telemetry to business KPIs with New Relic Pathpoint'
+summary: 'Get a strategic view of how your system performance impacts real-world business outcomes'
+releaseDate: '2024-10-10'
+learnMoreLink: 'https://docs.newrelic.com/docs/new-relic-solutions/business-observability/install-pathpoint/'
+getStartedLink: 'https://onenr.io/0oQDP60y7Qy'
+---
+
+New Relic has released a new version of Pathpoint, built on an entirely new code base featuring significant enhancements and AI capabilities. First introduced in January 2024, this version offers deeper business insights with a unique journey flow layout that visualizes stages, steps, and signals to create a shared understanding of business health for both frontline teams like customer support and higher-level stakeholders such as product managers, business analysts, and C-level executives.
+
+New features include:
+
+* **Intuitive UX:** No-code, entity-centric UX that allows you to instantly add any signal, alert, and service levels to your business flows to simplify business analysis (no need for custom NRQL).
+* **Multiple journey modeling:** The ability to model multiple business journeys per instance for deeper insights in complex environments.
+* **Playback mode:** Instantly track the state of your business stages over time for proactive issue resolution and to track improvements.
+* **Import/export with audit trails:** Easily move journeys between instances with an audit log for less toil and better version control.
+* **Automatically identify instrumentation gaps:** Close blind spots and have a more complete view of your digital business processes.
+* **AI-Driven Insights:** Fully integrated AI alerting and anomaly detection for real-time insights and fast root cause analysis.
```

Comment:
```suggestion
* **AI driven Insights:** Fully integrated AI alerting and anomaly detection for real-time insights and fast root cause analysis.
```

Reply:
Is this AI as in genAI, or are we referring to applied intelligence here? I'm asking because it's occurring in a statement about alerts and anomaly detection